packages/bpk-component-price/src/BpkPrice.module.scss

```
@@ -35,21 +35,27 @@
     text-decoration-line: line-through;
   }
 
-  &__titleColumn {
+  &__columnContainer {
```

Can this be, to follow BEM
```suggestion
  &__column-container {
```